Packs/CommonTypes/ReleaseNotes/1_9_0.md

```diff
@@ -0,0 +1,10 @@
+
+#### Incident Fields
+- **Account Name**
+- **Device Model**
+- **MD5**
```

Comment:
md5, sha256 and i think all of the rest are existing fields, @David-BMS could you help out here please?